readme.md

```diff
@@ -78,33 +78,37 @@ wide-contract-functions that result in exceptions being thrown but such
 behaviour is always well-defined. `std::vector` has `operator[]` and `at` to
 perform index-access with narrow and wide contracts respectively.
 
-### Changes to contracts
+### Changes to function contracts
```

Comment:
It's not just functions. Although the next paragraph sure makes it sound like it is...